application.js

```diff
@@ -108,7 +109,8 @@ function workerExitHandler(worker, code, signal) {
 // Start function
 // The number of workers to start can be specified with the FH_NUM_WORKERS env variable
 function start() {
-  if (cluster.isMaster) {
+  var starter = process.argv[1].split(path.sep).pop(); //check if app is being started by a test
+  if (cluster.isMaster && starter !== '_mocha') {
```

Comment:
`process.env.NODE_ENV === 'test'` works and is more readable. Adding this in. Thanks guys